core/src/main/java/org/apache/gravitino/GravitinoEnv.java

```diff
@@ -481,12 +482,12 @@ private void initGravitinoServerComponents() {
         new TopicNormalizeDispatcher(topicHookDispatcher, catalogManager);
     this.topicDispatcher = new TopicEventDispatcher(eventBus, topicNormalizeDispatcher);
 
-    // TODO(jerryshao). Add Hook and event dispatcher support for Model.
+    // TODO(jerryshao). Add Hook support for Model.
```

Comment:
@jerryshao , is `Hook` here equal to event dispatcher?

Reply:
Hook is another event listener made by @jerqi for authn related things.